ql/src/java/org/apache/hadoop/hive/ql/udf/ptf/ValueBoundaryScanner.java

```diff
@@ -760,6 +760,7 @@ public static SingleValueBoundaryScanner getBoundaryScanner(BoundaryDef start, B
     case "int":
     case "bigint":
     case "smallint":
+    case "tinyint":
```

Comment:
Good catch.  But can you also check if this type needs to be handled anywhere else for the window function code ? e.g line 730 above.